src/views/mfa-verify/PushForm.js

```diff
@@ -39,15 +39,24 @@ define(['okta', 'util/CookieUtil', 'util/Util'], function (Okta, CookieUtil, Uti
         function (state, isMfaRejectedByUser) {
           this.setSubmitState(isMfaRejectedByUser);
           if (isMfaRejectedByUser) {
-            this.showError(Okta.loc('oktaverify.rejected', 'login'));
+            this.showError({
+              responseJSON: {
+                errorSummary: Okta.loc('oktaverify.rejected', 'login')
+              }
+            });
           }
         }
       );
       this.listenTo(this.options.appState, 'change:isMfaTimeout',
         function (state, isMfaTimeout) {
           this.setSubmitState(isMfaTimeout);
           if (isMfaTimeout) {
-            this.showError(Okta.loc('oktaverify.timeout', 'login'));
+            this.showError({
+              name: Enums.MFA_VERIFY_ERROR,
```

Comment:
would it make more sense to have a value for session timeout? 
```
name: Enums.SESSION_TIMEOUT,
```
or 
```
name: Enums.SESSION_EXPIRED,
```
the reason I am asking is that an MFA_VERIFY_ERROR sounds to me that something went wrong while verifying a factor.